src/main/resources/localization/messages-nl.properties

```
@@ -0,0 +1,82 @@
+error.permissions.bot = Ik mis de volgende permissies: `Kanaal bekijken`, `Berichten verzenden`, `Berichtengeschiedenis bekijken`, `Ingesloten links`. Zorg ervoor dat deze permissies zijn ingesteld op dit kanaal, of voeg de bot opnieuw toe via [deze]({0}) link.
+error.invalid_time.format = Ik kan `{0}` niet omzetten naar een geldige tijdslengte!
+error.invalid_time.min = De duur die jij hebt opgegeven ({0}) was korter dan de minimum duur ({1})!
+error.invalid_time.max = De duur die jij hebt opgegeven ({0}) was langer dan de maximum duur ({1})!
+error.max_giveaways.guild = Momenteel lopen er al {0} giveaways in deze server!
+error.max_giveaways.channel = Momenteel lopen er al {0} giveaways in dit kanaal!
+error.invalid_winners.format = Ik kan `{0}` niet omzetten naar een geldige hoeveelheid winnaars!
+error.invalid_winners.max = De hoeveelheid winnaars die jij hebt opgegeven ({0}) is niet tussen 1 en ({1})!
+error.invalid_prize.length = De lengte van de prijs kan niet meer zijn dan {0} karakters!
+error.invalid_descr.length = De lengte van de beschrijving kan niet meer zijn dan {0} karakters!
+error.invalid_emoji.length = Eigen knoptekst kan niet langer zijn dan {0} karakters!
+error.invalid_emoji.choice = Volledige eigen giveawayknoppen zijn alleen beschikbaar met een Premium abonnement. Niet-leden kunnen alleen deze emoji''s gebruiken: {0}
+error.invalid_id = Ik kon `{0}` niet omzetten naar een berichten-ID!
+error.message_not_found = Ik kon geen bericht met het ID `{0}` vinden in dit kanaal!
+error.invalid_message = Dit bericht is geen compleet giveaway-bericht!
+error.giveaway_not_found =  Ik kon geen giveaway vinden met het ID `{0}`!
+error.giveaway_already_entered = Je doet al mee aan deze giveaway!
+error.giveaway_not_entered = Je doet nog niet mee aan deze giveaway!
+error.no_dms = Deze bot kan niet worden gebruikt in DM!
+error.giveaway_cooldown = Er is momenteel een cooldown voor het maken van een giveaway op deze server!
+error.giveaway_ended = Je kan bij deze giveaway niet meer meedoen of verlaten, want de giveaway is al geëindigd!
+error.invalid_color = Ik kon {0} niet omzetten naar een geldige kleur.
+error.reroll_message_option = Winnaars voor de giveaway kunnen opnieuw gekozen worden op de originele manier via: Rechter muisklik (Desktop) / Lang ingedrukt houden (Telefoon) > **Apps** > **Reroll giveaway**
+error.generic.ending = Er ging iets fout bij het beëindigen van deze giveaway.
+error.generic.delete = De giveaway is gestopt, maar er is iets foutgegaan tijdens het verwijderen van het giveaway-bericht.
+error.generic.creation = Er ging iets fout tijdens het maken van de giveaway.
+error.generic.enter = Er is iets fout gegaan tijdens het meedoen aan deze giveaway.
+error.generic.reroll = Er is iets fout gegaan tijdens het opnieuw kiezen van winnaars.
+error.generic = Er is iets fout gegaan.
```

Applies to the other generic errors too, since you use both grammar forms. For consistency's sake try to use one form.
```suggestion
error.generic = Er ging iets fout.
```